### DISCUSSION TITLE
Looking for recommendation on literature about anti-aliasing

### DISCUSSION THREAD

**Discussion by thor171** [Bodmer/TFT_eSPI#911] [General]
Hi,
this topic might be not directly related to this library, but I try to implement a simple anti-aliasing method, which then could be used with this lib.
Unfortunately, it's hard to find good books or papers about how different anti-aliasing methods works and how to try to implement them.

Maybe somebody can recommend something? This would be great! :-)

**Comment by Bodmer** [comment]
I think you will have to persist with Google and try to narrow down the search with suitable search words. For example "anti-aliasing graphics algorithm" or for software maybe "rosetta code c software github anti aliasing". It depends what you wish to do... AA on images is used a lot in the gaming graphics engines.

For simple line drawing look up Xiaolin Wu’s line algorithm, this approach can also be adapted for smoothing edges.  Look at the [examples here.](http://members.chello.at/~easyfilter/bresenham.html).

**Reply by thor171** [reply]
Thanks! I saw some implementations for openGL but I wanted to explore this topic more detailed.
I guess Xiaolin Wu's line algorithm is a very interesting approach and might be good for my application.
